release/release.js

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/env node
+const chalk = require('chalk')
+
+const config = require('./releaseConfig')
+const terminalUI = require('./utils/terminalUI')
+const io = require('./utils/IO')
+const file = require('./utils/file')
+const processes = require('./utils/processes')
+
+const { welcomeMessage, stepTitle } = terminalUI
+const { question, getNumberInput, proceedYesNo } = io
+const { spawnAssumeOkay, execAssumeOkay, execWithErrorHandling, exitRelease} = processes
+const { replaceInFile, readInFile } = file
+
+const { VERSION } = process.env
+
+const checkWorkspaceIsClean = async () => {
+  const { stdout: workspaceIsUnclean } = await execAssumeOkay('git diff-index --quiet HEAD -- || echo "not clean"')
+  if (workspaceIsUnclean) {
+    console.error('❌ Your git workspace must be clean before starting a release 🤖😞')
+    exitRelease()
+  }
+  config.write('safeToClearWorkspace', true)
+}
+
+const checkRequiredParams = () => {
+  const required = ['VERSION']
+  const missingEnvKeys = required.filter(reqEnv => !process.env[reqEnv])
```

Comment:
i think you could replace filter here with `every` and then use a boolean below

Reply:
I can see why filter is being used as it's good to inform what are the missing environment variables. 
But at the same time a `isMissingEnvKeys` boolean variable is cleaner code. Maybe assign an empty array check result to a variable with that name instead of just `missingEnvKeys.length`.